internal/routes/apollo/routes.go

```diff
@@ -80,15 +86,62 @@ func (a *Apollo) healthz(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	w.Write([]byte("OK"))
 }
 
+func (a *Apollo) getNamespace(appID string, cluster string, namespace string) (watcher.Namespace, error) {
+	for _, w := range a.w {
+		cm := w.Config()
+
+		// Trim off suffix .properties first
+		s := strings.TrimSuffix(namespace, ".properties")
+		for _, v := range cm {
+			ns, ok := v[cluster][s]
+			if ok {
+				return ns, nil
+			}
+
+			// Check if ns.properties exist
+			ns, ok = v[cluster][s+".properties"]
```

Comment:
do we really need to support a config file with and without .properties?

Reply:
I follow what apollo behave, it would return JSON when request with or without.properties